_client/client_app/angular_app/elements/iwish-app/iwish-app.html

```diff
@@ -0,0 +1,69 @@
+<link rel="import" href="../../bower_components/polymer/polymer.html">
+<link rel="import" href="../../bower_components/core-drawer-panel/core-drawer-panel.html">
+<link rel="import" href="../../bower_components/core-animated-pages/core-animated-pages.html">
+<link rel="import" href="../../bower_components/core-animated-pages/transitions/slide-up.html">
+<link rel="import" href="../../bower_components/core-animated-pages/transitions/slide-down.html">
+<link rel="import" href="../../bower_components/core-animated-pages/transitions/scale-up.html">
+<link rel="import" href="../../bower_components/core-animated-pages/transitions/tile-cascade.html">
+<link rel="import" href="../../bower_components/core-animated-pages/transitions/hero-transition.html">
+<link rel="import" href="../../bower_components/core-media-query/core-media-query.html">
+
+<link rel="import" href="../iwish-login-form/iwish-login-form.html">
+
+<polymer-element name="iwish-app" attributes="">
+  <template>
+    <link rel="stylesheet" href="iwish-app.css">
+
+  <core-animated-pages selected="{{selected}}" transitions="cross-fade cross-fade-delayed scale-up slide-up slide-up-offscreen slide-down tile-cascade hero-transition">
+    <section>
+      <iwish-login-form layout vertical center center-justified slide-down response="{{out}}"></iwish-login-form>
+    </section>
+    <section>
+      <iwish-login-form layout vertical center center-justified slide-down></iwish-login-form>
+    </section>
+  </core-animated-pages>
+
+    <paper-button raised on-click="{{stuff}}">Next view in app</paper-button>
+
+  </template>
+  <script>
+
+
+    Polymer('iwish-app', {
+      selected: 1,
+
+         change : function () {
+      var s = document.querySelector('select');
```

Comment:
please rename this variable to something more descriptive. I can't understand what this element is...